tests/methods/ParseTest.php

```diff
@@ -157,6 +157,31 @@ public function testGetColumnDatatypes() {
         $this->assertEquals($expected, $this->csv->data_types);
     }
 
+
+    public function testDataArrayKeysWhenSettingOffsetWithHeading() {
+        $this->csv->offset = 2;
+        $this->csv->auto(__DIR__ . '/fixtures/datatype.csv');
+        $expected = [
+            'title',
+            'isbn',
+            'publishedAt',
+            'published',
+            'count',
+            'price'
+        ];
+
+        $this->assertEquals($expected, array_keys($this->csv->data[0]));
```

Comment:
What is happening here? From a distance I would have expected:
- the `sep=;` row gets ignored
- 2 additional rows get ignored; because ``$heading`` is true by default, the remaining data row "Weißwein" is returned

I probably miss something obvious here.

Reply:
Indeed, I missed the `array_keys` call in there. Perfect!